### PR TITLE
added initial set up for functional test more to come

### PR DESCRIPTION
**Pull request by glambert22**
Functional Tests...
adding inital setup for selenium test. One test passed. There other for the dropdowns needs some work but it's late and I need some shut eye. 
- [x] Created initial test to test page load
- [x] Create initial test to select date dropdown
- [ ] Create better css selectors for dropdowns
- [ ] Select Date and State

| Jenkins Enabled Test | Dockerized Tests |
| --- | --- |
| initial page load | initial page load |
| correct selectors | test combined CI container |

### CODE REVIEW

File: functional_tests/tests/smoketests/happyPath.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+            .assert.title('Aurora')
+            .assert.containsText('body', 'Track food threats near you')
+            .pause(1000)
+            .assert.containsText("button[data-toggle=\"dropdown\"] ", "Select State")
+            .click("#crit-navbar .dropdown-toggle")
+            .pause(1000)
+            //.assert.containsText("button[data-toggle=\"dropdown\"] ", "Select State")
+            .end();
+    }
+};
```
Comment (jasonmullins) [inline functional_tests/tests/smoketests/happyPath.js:15]: newline at eof

**Comment by jasonmullins** [comment]
new line and then :thumbsup:

**Comment by krullshards** [comment]
Jason, the newline nazi.

**Comment by jasonmullins** [comment]
You'll thank me if we ever start using js_lint or something.

**Comment by jasonmullins** [comment]
I accidentally merged it... ha.  Ok, I'll fix the newline in my branch.

**Comment by glambert22** [comment]
yeah I both js_hint/js_lint will scream at you w/o I can added.. I'm going to write  more test today but wanted to talk with you about our css selectors

**Comment by jasonmullins** [comment]
Ok. if you need to add ids or classes for testing - go for it.

**Pull request merged**